source/drivers/MPU6050.cpp

```diff
@@ -89,23 +89,17 @@ int MPU6050::updateSample()
     return DEVICE_OK;
 };
 
-int MPU6050::setRange(int range)
-{
-    sampleRange = range;
-    return configure();
-}
-
-Sample3D MPU6050::getSample()
-{
-    return sample;
-}
-
 void MPU6050::idleCallback()
 {
 
 }
 
-MPU6050::~MPU6050()
+int MPU6050::setSleep(bool sleepMode)
 {
-}
+    if (sleepMode)
+        return 0; // TODO

```

Comment:
```suggestion
        return i2c.writeRegister(address, 0x6B, 0x40);
```

I think that should do the trick